gamemode/modules/hitmenu/sh_init.lua

```diff
@@ -30,6 +30,7 @@ DarkRP.getHitmanTeams = fp{fn.Id, hitmanTeams}
 function DarkRP.hooks:canRequestHit(hitman, customer, target, price)
     if not hitman:isHitman() then return false, DarkRP.getPhrase("player_not_hitman") end
     if customer:GetPos():DistToSqr(hitman:GetPos()) > minHitDistanceSqr then return false, DarkRP.getPhrase("distance_too_big") end
+    if not customer:Alive() then return false, Darkrp.getPhrase("must_be_alive_to_do_x", DarkRP.getPhrase("place_a_hit")) end
```

Comment:
Did you test this? `Darkrp` is not the correct casing.

Reply:
Whoops, I think I fixed that in my test build, but forgot to fix it here. Pretty glaring mistake haha. Good catch